### DISCUSSION TITLE
Multiple outputs from one Controller input

### DISCUSSION THREAD

**Discussion by ICE753** [darthcloud/BlueRetro#675] [Q&A] [answered]
Was just wondering if it's possible at all currently to have one controller control all four outputs of my blueretro? For example just having one controller connected and four gamecube output cables each plugged into a separate gamecube in order to control them all at the same time with identical inputs?

**Comment by darthcloud** [comment, marked as answer]
Yes you can, you need to add duplicate mapping of all buttons in first mapping and set the output to the other ports.

**Reply by darthcloud** [reply]
But make sure only one GC provide power to blueretro, the other three shall have only data and GND connection.

**Reply by ICE753** [reply]
Awesome, thank you so much :)
